README.md

```diff
@@ -162,6 +162,14 @@ directory and add some aliases to `PATH`.
    ```
    These commands would re-generate project files with coverage enabled and run tests. Coverage report would be in `build/__coverage`.
 
+3. Create SkyrimPlatform archive with structure identical to our Nexus uploads (Windows-only):
```

Comment:
I know it's located inside of optional steps section, but I think that it should be clarified somehow of why one could want to to this. Something like 'If you wish to use solely SP build, you can run this script to get zip with only related files'

And a few thoughts more (out of scope of this PR):
1. Always packing files look unnecessary, I guess a frequent case would be 'build -> create zip from **some of** dist files -> unpack zip -> use it in MO2', so...
2. ... probably it makes more sense to make a script that copies files to a separate folder instead (or just deletes extra files in place)
3. Or, ideally, having an option to run some target that will only add those files to `build/dist` and nothing more